jcl/src/java.base/share/classes/jdk/internal/nicl/NativeInvoker.java

```diff
@@ -175,5 +183,73 @@ private static long getStructAddress(Reference<?> arg) throws IllegalAccessExcep
 		Pointer<?> ptr = RuntimeSupport.createPtr(addr, returnLayoutType);
 		return (Reference<?>) ptr.deref();
 	}
+	
+	private static long generateFunctionPointerBinding(Object anonInstance) {
+		Class<?> lambdaClass = anonInstance.getClass();
+		Method method = null;
+		Method[] arrayOfMethod;
```

Comment:
Why is `arrayOfMethod` assigned on the next line (in the middle of another expression) rather than on this line?

Reply:
Couldn't this be `arrayOfMethod = lambdaClass.getDeclaredMethods();` and `j` could be assigned as `int j = arrayOfMethod.length;`?